README.md

```diff
@@ -13,22 +13,20 @@ HTML:
 Graphviz:
 ![nmap-example-graphviz](docs/images/example-dot.png)
 
-A tool that allows you to convert NMAP XML output to html/csv/json/markdown/dot/sqlite.
+A tool that allows you to convert NMAP XML output to excel/html/csv/json/markdown/dot/sqlite.
 
 ## Installation
 
-It's possible to install it using `go install` command
-
 ```
-go install github.com/vdjagilev/nmap-formatter/v2@latest
+git clone https://github.com/gorkavp/nmap-formatter.git
+cd nmap-formatter
+go build
 ```
 
-All other options can be found on [Installation Wiki page](https://github.com/vdjagilev/nmap-formatter/wiki/Installation).
-
```

Comment:
:thinking: I think people should be able to visit the wiki installation just by clicking, not sure why we are removing this.